cmd/rook/rook.go

```diff
@@ -67,3 +57,22 @@ func addCommands() {
 func NewTableWriter(buffer io.Writer) *tabwriter.Writer {
 	return tabwriter.NewWriter(buffer, outputMinWidth, outputTabWidth, outputPadding, outputPadChar, 0)
 }
+
+func enableLogging() {
+	debugDir := os.Getenv(debugLogVar)
```

Comment:
does this have the side effect that all our internal log.Printf statments will now go to stdout by default?  for example, the goal of a command like `node ls` is to just have the pretty table printed out, not to have a bunch of log statements as well.